src/Item/PluginItem/Interpretations/Interpretation.js

```diff
@@ -118,7 +130,7 @@ class Interpretation extends Component {
     };
 
     showCommentField = () => {
```

Comment:
The use of this has changed, so it would be good to rename it accordingly (e.g., toggleCommentField)